functests/utils/mcps/mcps.go

```diff
@@ -52,6 +55,18 @@ func GetByName(name string) (*machineconfigv1.MachineConfigPool, error) {
 	return mcp, err
 }
 
+// GetByProfile returns the MCP by a given performance profile
+func GetByProfile(performanceProfile *performancev1alpha1.PerformanceProfile) (string, error) {
+	mcpLabel := profile.GetMachineConfigLabel(performanceProfile)
+	key, value := components.GetFirstKeyAndValue(mcpLabel)
```

Comment:
If we have more of this, is it an error?

Reply:
no error since its getting only the First key and value. (maybe this should be more generic but it should be enough for the tests atm)

Reply:
Why not fail? I don't like the approach "take the first one" since it looks like wishful thinking (i this case I wish the first one will pass get the test to pass :) )